rules/enforce-svg-import.js

```diff
@@ -0,0 +1,41 @@
+const reportError = (context, node) => {
+  const parts = node.source.raw.split(".")
+  const name = parts[1]
```

Comment:
Hard-coding `parts[1]` might lead to issues when you have a filepath like `'../../../../../svg/Christiaan.svg'`. Can't we use anything other then regex here to get the filename of the include? Or have a regex that supports the case above.